### PR TITLE
LPS-200285

### PR DESCRIPTION
**Pull request by JhossephG**
https://liferay.atlassian.net/browse/

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:object-functional

**Comment by evanilson-santana** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:object-functional

File: ci.properties [hunk 1]
```diff
@@ -329,6 +330,7 @@ ci.test.suite.description[object-as]=Test object tests with different applicatio
 ci.test.suite.description[object-db]=Test object tests with different databases.
 ci.test.suite.description[object-db-partition]=Test object functional tests with Database Partitioning.
 ci.test.suite.description[object-lxc]=Test Object tests with Liferay Experience Cloud configurations.
+ci.test.suite.description[object-functional]=Test Object tests with only functional tests.
```
Comment (timpak) [inline ci.properties:333]: Please follow the existing patterns for the descriptions.

**Review by timpak — CHANGES_REQUESTED** [review]
@JhossephG 68e89fbff43f8d64391f52fc1456e4bff2abd65f is two separate commits, one to add the new test suite, another to sort the `object-lxc` test suite. Please split them into two. Use the commit message `Sort` when sorting code.

Also, please use the correct tense for your commit message. It shouldn't be `Adding`, it should be `Add`. This isn't the first time I had to correct this so please ensure this doesn't happen again moving forward.

**Comment by JhossephG** [comment]
> @JhossephG [68e89fb](https://github.com/liferay-bpm-qa/liferay-portal/commit/68e89fbff43f8d64391f52fc1456e4bff2abd65f) is two separate commits, one to add the new test suite, another to sort the `object-lxc` test suite. Please split them into two. Use the commit message `Sort` when sorting code.
> 
> Also, please use the correct tense for your commit message. It shouldn't be `Adding`, it should be `Add`. This isn't the first time I had to correct this so please ensure this doesn't happen again moving forward.

The SF that changed the sorting, so the commit name should be SF? I not separate into a different commit since we don't usually split SF commits

File: ci.properties (hunk 1 above)
Comment (JhossephG) [reply ci.properties:333]: I'll follow the same pattern from `commerce-functional` then.

**Comment by JhossephG** [comment]
ci:forward

**Pull request closed without merging**